### ISSUE TITLE
git Ignored files are not distinguished in sidebar

### ISSUE THREAD

**Issue by rchl** [SublimeText/Theme-DAneo#3]
Default theme makes git-ignored files stand out in the side bar by making text color a little darker.
![Screenshot 2019-07-10 at 22 07 40](https://user-images.githubusercontent.com/153197/61001317-b24b2300-a35f-11e9-92df-a1aebc43ea86.png)

DAneo seems to be missing that distinction.

**Comment by deathaxe** [comment]
I saw different approaches to distinguish git states by text colors, but found it strange somehow. But with the option to disable badges, supporting optional colors is indeed a reasonable request. Should not be too difficult.

**Comment by rchl** [comment]
Do badges really play a role here? I think badges won't show up for those git-ignored files in any case.

**Comment by deathaxe** [comment]
Ah, right. Didn't add badges for ignored files.

**Comment by deathaxe** [comment]
Should be fixed by v0.2.0

Also added support for colored sidebar labels based on vcs status, which can be controlled by following settings in the _Preferences.sublime-settings_.

```

	// Controls whether to show vcs status of ignored files and folders
	// by the color of their labels in the sidebar.
	"show_sidebar_ignored_label_colors": true,

	// Controls whether to show vcs status of tracked files and folders
	// by the color of their labels in the sidebar.
	"show_sidebar_tracked_label_colors": true,

	// Controls whether to show vcs status of untracked files and folders
	// by the color of their labels in the sidebar.
	"show_sidebar_untracked_label_colors": true,
```

**Comment by rchl** [comment]
I love it. Thanks. :)
